### ISSUE TITLE
[FEATURE] getOfferingsForSKU and updateOfferings

### ISSUE THREAD

**Issue by aabrodskiy** [etsy/open-api#33]
**Is your feature request related to a problem? Please describe.**
Current way of getting and updating inventory is requiring to fetch all listings first with entire structure, then submit the structure back to update the inventory. This is particularly painful for multi-variant listings.
My use cases involve external inventory management, where I need to update inventory for certain SKUs only. Same SKUs could be sold as variants in multiple listings.

**Describe the solution you'd like**
I would like a simple way to get all offerings for one sku (**getOfferingsForSKU**) or for a list of sku's (**getOfferingsForSKUs**). This would allow to fetch directly all offerings without getting through listing->product->property on value ->offering path and would return all offerings required.
Then the second call would be **updateOfferings** for one or many offerings. 

For **updateOfferings**, the body would be:
[
 {
   offering_id,
   quantity,
   is_enabled,
   is_deleted,
   price
 }
]

This same above structure would be the response of **getOfferingsForSKUs** and **getOfferingsForSKU**

**Describe alternatives you've considered**
Alternative solution is much slower and requires many more steps:
1. Call getListings to get all listings, implement pagination if many
2. Call getListingInventory for each listing to get down to variants and their skus
3. Find all products with SKUs in question
4. call updateListingInventory for each of those products, also having to update products, which don't need to be updated (e.g. other variants in the listings, whose inventories haven't changed)

**Comment by foxfriends** [comment]
Yes a way to get/update by SKU would be useful, as most external software will be managing things on a SKU level.

The challenge is that SKUs, to my knowledge, are not necessarily unique or even related to an inventory level. The way that variations can optionally change price or change inventory or change SKU makes the inventory, pricing, and SKUs system unusually complicated... not sure what an API update could do to help with that, but this is one of the weirdest parts of the existing API I have encountered yet.
